data/mods/BombasticPerks/perks.json

```diff
@@ -716,7 +716,7 @@
     "id": "perk_eating_window",
     "name": { "str": "Compressed Eating Window" },
     "points": 0,
-    "description": "You're so used to squeezing all your meals into a short time that your body has adapted to it.  You gain +150% kcal and vitamins from any meals eaten between noon and 6 p.m., but 0 calories and vitamins for any meals eaten outside that time period.",
+    "description": "You're so used to squeezing all your meals into a short time that your body has adapted to it.  You gain +250% kcal and vitamins from any meals eaten between noon and 6 p.m., but 0 calories and vitamins for any meals eaten outside that time period.",
```

Comment:
This is misleading: it implies a 250% *bonus*, for a total of 350% caloric and vitamin content. The perk actually does add 150% to those, for a total of 250%.

I think "You gain 250% <...>", without the plus, should relate the idea clearly.

Reply:
Would `2.5 times` work better here? Because i saw people who assumed 150% means x0.5 bonus to calories

Reply:
That should work better. I suspect someone will still stumble over "2.5 times" as "*extra* 250%", because users are human, but at that stage, there's little the developers can do to make it any clearer.

Reply:
```suggestion
    "description": "You're so used to squeezing all your meals into a short time that your body has adapted to it.  You gain 2.5 times as much calories and vitamins from any meals eaten between noon and 6 p.m., but 0 calories and vitamins for any meals eaten outside that time period.",
```

Reply:
Should probably me "as many calories", since both calories and vitamins are countable.